### PR TITLE
Consolidate synthetics and uptime content

### PR DESCRIPTION
**Pull request by EamonnTP**
### Summary

This PR groups the Synthetic monitoring and Uptime Monitoring content under a new **Synthetic and Uptime monitoring** bucket. We may want to name this bucket just Synthetic monitoring?

The updates include:
- A new landing page with a brief intro to both [Synthetics and Uptime monitoring](https://observability-docs_1171.docs-preview.app.elstc.co/guide/en/observability/master/monitor-uptime-synthetics.html). 
- New content and landing pages for [Synthetic monitoring](https://observability-docs_1171.docs-preview.app.elstc.co/guide/en/observability/master/synthetic-monitoring.html) and for [Uptime monitoring](https://observability-docs_1171.docs-preview.app.elstc.co/guide/en/observability/master/monitor-uptime.html).
- Synthetics monitoring landing page renamed in the TOC to **Real Browser synthetic monitoring**.
- Uptime monitoring landing page renamed in the TOC to **Monitor uptime and services using ICMP, TCP, and HTTP checks**.
- Quick starts have now been renamed to **Get started using** topics in the TOC.
- Moved the **How does it work?** section to the main [**Get started using Docker**](https://observability-docs_1171.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-quickstart.html) topic. 

**Note**: In a follow-up PR, I would like to add a **Get started using Elastic Agent and Fleet** to the **Monitor uptime and services using ICMP, TCP, and HTTP checks** bucket.

### Docs preview

https://observability-docs_1171.docs-preview.app.elstc.co/guide/en/observability/master/monitor-uptime-synthetics.html

### Related issues

https://github.com/elastic/observability-docs/issues/1004
Closes https://github.com/elastic/observability-docs/issues/1170
Closes https://github.com/elastic/observability-docs/issues/1189

### CODE REVIEW

**Comment by EamonnTP** [comment]
Hi @andrewvc If you have time this week, could I get a final review? Thanks!

File: docs/en/observability/monitor-uptime.asciidoc [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+host is available and connected to the network. However, it doesn't tell you if a service on the host is running or
+not.
+
+| *TCP monitor* | Monitor the services running on your hosts. The TCP monitor checks individual ports
+to ensure the service is accessible and running.
+
+| *HTTP monitor* | Monitor your website. The HTTP monitor checks to ensure specific endpoints return the correct
+status code and display the correct text. 
+
+|===
```
[the remark this answers is not in the document]
Comment (andrewvc) [reply docs/en/observability/monitor-uptime.asciidoc:28]: I agree, 100% @paulb-elastic

[the remark this answers is not in the document]
Comment (andrewvc) [reply docs/en/observability/monitor-uptime.asciidoc:28]: To add a little more detail, I think that structurally this page is a vestige of the old organization. If this is only going to cover the lightweight checks we should title the page "Lightweight Synthetics Checks"

I think this page is supposed to center on the Uptime app, which monitors both browser and lightweight checks, but isn't especially geared toward lightweight vs. heavy checks, it does the entirety of both cases

File: docs/en/observability/synthetics-quickstart-fleet.asciidoc [hunk 2]
```diff
@@ -76,6 +80,6 @@ See <<synthetics-visualize>> for more information.
 [[synthetics-quickstart-fleet-security]]
 == Note on Sandboxing and Security
 
-Synthetics Browser checks can only run as the non-root `elastic-agent` user  in the `elastic-agent` container. Synthetics ICMP checks, on the other hand, require running as the root user. We are currently working to simplify the permissions requirements to work with a single user. This can be tracked in https://github.com/elastic/beats/issues/27920[this github issue].
+Synthetics Browser checks can only run as the non-root `elastic-agent` user  in the `elastic-agent` container. We are currently working to simplify the permissions requirements to work with a single user. This can be tracked in https://github.com/elastic/beats/issues/27920[this github issue].
```
Comment (andrewvc) [inline docs/en/observability/synthetics-quickstart-fleet.asciidoc:83]: Thankfully this is no longer true in 7.16+! We can remove this caveat.

File: docs/en/observability/monitor-uptime.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrewvc) [reply docs/en/observability/monitor-uptime.asciidoc:28]: OK, I think I see what's going on here now, I strike the last half of my second comment. The changes that probably make the most sense are:

1. Change the title to "Lightweight synthetic monitoring"
1. Rename this file to `monitor-lightweight.asciidoc`

We should be attempting to phase out the word "Uptime" everywhere possible, with the one obvious exception being the "Uptime app". We shouldn't refer to "Uptime checks" etc.

[the remark this answers is not in the document]
Comment (EamonnTP) [reply docs/en/observability/monitor-uptime.asciidoc:27]: Created https://github.com/elastic/observability-docs/issues/1232 to address in a followup PR.

**Pull request merged**